policy/modules/services/sendmail.te

```diff
@@ -173,6 +173,7 @@ optional_policy(`
 ')
 
 optional_policy(`
+	userdom_use_user_ttys(sendmail_t)
```

Comment:
To confirm, this is only related to posfix use? Also, is it truly limited only to ttys?

Reply:
It should apply to the sendmail program from every MTA which can be run from the command line and often is.

Reply:
If it's not specific to postfix use, then it shouldn't be in the optional with postfix rules.

`userdom_use_inherited_user_terminals()` would cover ttys and ptys, plus prevent it from opening arbitrary terminals.

Reply:
done